plugin.php

```diff
@@ -56,8 +56,10 @@ function ldapauth_environment_check()
 	return true;
 }
 
-
-yourls_add_filter('is_valid_user', 'ldapauth_is_valid_user');
+# Reroute login to yourls filter
+# (see https://github.com/YOURLS/YOURLS/wiki/Advanced-Hook-Syntax)
+//yourls_add_filter( 'is_valid_user', 'ldapauth_is_valid_user' );
+yourls_add_filter('shunt_is_valid_user', 'ldapauth_is_valid_user');
```

Comment:
NOTE: I need to review the security implications of shunt_is_valid_user before I am comfortable merging this PR.